src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -265,7 +266,7 @@ private void apply(RoutingContext routingContext) {
           .mapTo(Location::from)
           .whenNotFound(failed(new ServerErrorFailure("Can`t find location")))
           .fetch(request.params().get(LOCATION_ID_NAME))
-          .thenCompose(r -> r.after(location -> getPolicyId(request.params(), drools, location)))
+          .thenCompose(r -> r.after(location -> getPolicyIdAndRuleLine(request.params(), drools, location)))
```

Comment:
I don't really understand why we return the line number, rather than more information about the rule that matched, e.g. information about which parameters were a match.

Reply:
this code is outdated for now